webpack.config.js

```diff
@@ -9,13 +9,28 @@ var base = {
         host: '0.0.0.0',
         port: process.env.PORT || 8073
     },
-    devtool: 'source-map',
+    devtool: 'cheap-module-source-map',
+    module: {
+        rules: [
+            {
+                // allow ES2015 in any *.js file under .../scratch-*/src/...
+                test: /[\\/]+scratch-[^\\/]+[\\/]+src[\\/]+.+\.js$/,
+                loader: 'babel-loader',
+                options: {
+                    presets: ['es2015']
+                }
+            }
+        ]
+    },
     plugins: [
         new webpack.optimize.UglifyJsPlugin({
             include: /\.min\.js$/,
             minimize: true
         })
-    ]
+    ],
+    resolve: {
+        symlinks: false
+    }
```

Comment:
This affects the way webpack looks for plugins when dealing with `npm link`ed dependencies (say, `scratch-render` for example). If `symlinks` is true (default) then webpack will look for `scratch-render/node_modules/` only and will fail if the plugin is not found there. If `symlinks` is false it will find the plugin one `scratch-vm/node_modules/...`
See https://github.com/webpack/webpack/issues/985 for multiple pieces of conflicting advice. This setting works for me but may need further tinkering to make sure everything is correct.